cmd/internal/shared/identityserver/config.go

```diff
@@ -28,15 +28,17 @@ import (
 var DefaultIdentityServerConfig = identityserver.Config{
 	DatabaseURI: "postgresql://root@localhost:26257/ttn_lorawan_dev?sslmode=disable",
 	OAuth: oauth.Config{
+		Mount: "/oauth",
 		UI: oauth.UIConfig{
 			TemplateData: webui.TemplateData{
 				SiteName:      "The Things Stack for LoRaWAN",
+				Title:         "Account",
 				Language:      "en",
 				CanonicalURL:  shared.DefaultOAuthPublicURL,
 				AssetsBaseURL: shared.DefaultAssetsBaseURL,
 				IconPrefix:    "oauth-",
-				CSSFiles:      []string{"oauth.css"},
-				JSFiles:       []string{"oauth.js"},
+				CSSFiles:      []string{"account.css"},
+				JSFiles:       []string{"account.js"},
```

Comment:
This is a breaking change for deployments that use custom JS/CSS files (which I don't think is very common). In any case, make sure to mention in CHANGELOG that deployments need to change their config if they customized these.